<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowContainer</class>
 <widget class="QWidget" name="WindowContainer">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="windowTitle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="detachButton"/>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
